### ISSUE TITLE
Support cyrillic letters

### ISSUE THREAD

**Issue by Xaxoxuxu** [romgere/text2stl#72]
There is no obvious way to choose a font (if there even is any) that can render Cyrillic in the preview and afterward generate the STL

**Comment by romgere** [comment]
@Xaxoxuxu can you provide some font / example of what you're trying to achieve, I have no idea how to test Cyrillic rendering, so some help will be appreciate.

**Comment by Xaxoxuxu** [comment]
Hi, @romgere
Let's say we want to generate a model of the word "ТЕСТ" which is "TEST" in Bulgarian. This is what we get: https://imgur.com/a/JHgxKSE
The letters are replaced with squares.

**Comment by romgere** [comment]
@Xaxoxuxu are you using the default font here ?

**Comment by Xaxoxuxu** [comment]
@romgere yes, everything default in here: https://text2stl.mestres.fr/en-us/generator
Also tried with 20 different fonts, and every time its gibberish or squares.
I believe you can reproduce it with АСДФГ and have a look

**Comment by romgere** [comment]
Thx @Xaxoxuxu I can reproduce, I'll try to dig into this soon.

**Comment by romgere** [comment]
@Xaxoxuxu I did some recent update (mainly bug fix) on the tool & the default font is now "Roboto"...

With this font it seems Cyrillic is working : 
![image](https://user-images.githubusercontent.com/13900970/220411831-2cd56c43-6703-4c45-ad4d-c8fb535440f3.png)


I think your issue is because of some font are not compatible with Cyrillic... The tool probably need some additional options in font tab to allow users to filter font by "Script" (arabic, cyrillic, latin, ...).

In waiting for a such improvement, you can probably use [google font site](https://fonts.google.com/?subset=cyrillic&noto.script=Cyrl) to find compatible font & try to find them in the tool. (or download them locally & use them as "custom font")

**Comment by romgere** [comment]
@Xaxoxuxu You should now be able to filter font by "language" : 
![image](https://user-images.githubusercontent.com/13900970/220650759-d3b2a9aa-5f56-4bf2-882c-1020c932c337.png)

I tested "АСДФГ" string with some font with Cyrillic support & didn't find issue...

Hope this will help users to find compatible font.

Feel free to re-open the issue is needed

**Comment by Xaxoxuxu** [comment]
Thank you, it works!
